src/thermo/src/4C_thermo_timint.hpp

```diff
@@ -561,7 +561,7 @@ namespace Thermo
     };
 
     std::shared_ptr<Core::IO::DiscretizationWriter> output_;  //!< binary output
-    std::shared_ptr<Core::IO::DiscretizationVisualizationWriterMesh>
+    std::optional<std::shared_ptr<Core::IO::DiscretizationVisualizationWriterMesh>>
```

Comment:
```suggestion
    std::optional<Core::IO::DiscretizationVisualizationWriterMesh>
```

Sorry if that was not clear before. This is what I meant.

Reply:
Done.